packages/enzyme-test-suite/test/ShallowWrapper-spec.jsx

```diff
@@ -2440,6 +2440,70 @@ describe('shallow', () => {
     });
   });
 
+  describe('.invoke(propName, ..args)', () => {
+    it('can update the state value', () => {
+      class Foo extends React.Component {
+        constructor(props) {
+          super(props);
+          this.state = {
+            count: 0,
+          };
+        }
+
+        render() {
+          const { count } = this.state;
+          return (
+            <div>
+              <button
+                type="button"
+                onClick={() => this.setState(state => ({ count: state.count + 1 }))}
+              >
+                {count}
+              </button>
+            </div>
+          );
+        }
+      }
+      const wrapper = shallow(<Foo />);
+      wrapper.find('button').invoke('onClick');
+      expect(wrapper.state('count')).to.equal(1);
+    });
+    it('can return the handlers return value', () => {
+      const spy = sinon.stub().returns(123);
+      class Foo extends React.Component {
+        render() {
+          return (
+            <div>
+              <a onClick={spy}>foo</a>
```

Comment:
Maybe change this to a button. There's a real risk that somebody will look at this code and use it as an example - and then are stuck with bad HTML semantics and potentially an app that doesn't work (don't think <a> tags are clickable without href)